client/src/lib.rs

```diff
@@ -47,13 +47,14 @@ impl ClientContract {
 
     /// symbol: Base64-encoded token symbol
     #[allow(dead_code)] // This function gets called from the oracle
-    pub fn demo_token_price(&mut self, symbol: String, spec_id: Base64String) {
+    pub fn demo_token_price(&mut self, symbol: String, spec_id: Base64String) -> U128 {
```

Comment:
Please rename this function and remove the 'demo' word from it. Let's start to think of this client contract as more than just for the demo.